core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -161,7 +162,9 @@ public static FileSystem create(FileSystemContext context) {
     public static FileSystem create(FileSystemContext context, FileSystemOptions options) {
       AlluxioConfiguration conf = context.getClusterConf();
       checkSortConf(conf);
-      FileSystem fs = new BaseFileSystem(context);
+      FileSystem fs = options.getUfsFileSystemOptions().isPresent()
```

Comment:
can this be used for a normal alluxio client?

Reply:
Currently it's the FileSystem options and is only supported to be enabled via FUSE, but can be coded to enable in other paths.
Not able to used via configuration through other alluxio clients yet